bird/ibgp_rrs.conf

```
@@ -0,0 +1,62 @@
+template bgp ibgp_peers {
+    local as LOCAL_ASN;
+    source address SERVE_IP;
+    graceful restart on;
+    # bfd on;

```

```suggestion
    # bfd on;
    add paths rx;
```
需要 add paths，将选路判断权利交给目标路由器

---

不需要